src/Table/Table.test.tsx

```diff
@@ -499,9 +499,7 @@ describe("Component: Table", () => {
             results = rows;
         });
 
-        const onRemoveFilter: jest.Mock = jest.fn((filterItem: FilterItem) => {
-            console.log(filterItem);
-        });
+        const onRemoveFilter: jest.Mock = jest.fn((filterItem: FilterItem) => true);
```

Comment:
and this

Reply:
Done

Reply:
you should resolve the conversation if it's done. KR or Yousif